pom.xml

```diff
@@ -246,7 +246,8 @@
         <maven-war-plugin.version>2.2</maven-war-plugin.version>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
         <maven-checkstyle-plugin.version>3.0.0</maven-checkstyle-plugin.version>
-
+	<maven-jacoco-plugin.version>0.8.1</maven-jacoco-plugin.version>
```

Comment:
✏️ Can you indent this so it's in line with the rest? Also looks like there are new lines above and below it.

Reply:
👍 I would say that this is the problem with no official formatters, you end up with difference in files 😞